### DISCUSSION TITLE
Vercel removes .html extension without cleanUrls vercel.json and leads to 404 errors

### DISCUSSION THREAD

**Discussion by defeated72** [vercel/community#681] [Help]
I have a bunch of client websites that do not have an vercel.json configured that just wants to use with .html extension.

When I look my pages at github I can see within the pages the links are like this:

<a href="test.html"

On the Vercel Website itself I see this:

<a href="test" which is wrong and gives all 404 errors.

What can I do?

**Comment by amyegan** [comment]
Hi @defeated72. Can you share the repo you're trying to deploy and the site where you are seeing the problem? If you can't share the repo then any other details about it, like what framework you're using, would help me look into this for you.
